sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/models/_patch.py

```diff
@@ -8,6 +8,23 @@
 """
 from typing import List
 
+from enum import Enum
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class SchemaFormat(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
Seems like we need to add this to the `__all__`?

Reply:
This is currently public under the azure.schemaregistry namespace, so I didn't add it to all here, since that would move it to azure.schemaregistry.models.